goodkey/blocked.go

```diff
@@ -0,0 +1,70 @@
+package goodkey
+
+import (
+	"crypto"
+	"errors"
+	"io/ioutil"
+
+	"github.com/letsencrypt/boulder/core"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+// blockedKeys is a type that maintains a map of Base64 encoded SHA256 hashes of
+// DER encoded PKIX public keys that should be considered blocked. blockedKeys
+// are created by using loadBlockedKeysList.
+type blockedKeys struct {
```

Comment:
Random style thing: (feel free to ignore, just a suggestion) you can do `type blockedKeys map[string]bool` and then directly define methods on it instead of wrapping it in a struct, which strips a little boilerplate. But there is also no _real strong_ reason not to do what you currently have, ¯\\\_(ツ)_/¯

Reply:
good idea, I like your version better :+1: Implemented in f95c9a0